test/pylib/scylla_cluster.py

```diff
@@ -402,17 +402,31 @@ async def start(self, api: ScyllaRESTAPIClient, expected_error: Optional[str] =
         sleep_interval = 0.1
         cql_up_state = CqlUpState.NOT_CONNECTED
 
+        def read_last_line(file_path: pathlib.Path):
+            max_line_bytes = 256
+            file_size = os.stat(file_path).st_size
+            with file_path.open('rb') as f:
+                f.seek(max(0, file_size - max_line_bytes), os.SEEK_SET)
+                line_bytes = f.read()
+            line_str = line_bytes.decode('utf-8', errors='ignore')
```

Comment:
Why are you not using readlines() here?

Reply:
it does not simplify the code much, we still need to open the file in binary mode to make seek work (https://docs.python.org/3/tutorial/inputoutput.html#methods-of-file-objects) and then decode bytes with `errors='ignore'`.